pkg/tfvars/ovirt/ovirt.go

```diff
@@ -0,0 +1,36 @@
+// Package ovirt contains ovirt-specific Terraform-variable logic.
+package ovirt
+
+import (
+	"encoding/json"
+)
+
+type config struct {
```

Comment:
only set the optional values as omitempty, all required values must always be set.

Reply:
done